pkg/storegateway/bucket.go

```diff
@@ -130,6 +129,12 @@ type BucketStore struct {
 	postingsStrategy postingsSelectionStrategy
 }
 
+type noopShapshotter struct {
+	services.Service
+}
+
+func (noopShapshotter) RestoreLoadedBlocks() map[ulid.ULID]int64 { return nil }
```

Comment:
I believe, now we don't need to implement the `RestoreLoadedBlocks` method, and it can be just a noop-service.